library/src/main/java/com/meisolsson/githubsdk/model/IssueEvent.java

```diff
@@ -26,9 +26,27 @@
 import com.squareup.moshi.Moshi;
 
 import java.util.Date;
+import java.util.List;
 
 @AutoValue
 public abstract class IssueEvent implements Parcelable {
+    @AutoValue
+    public static abstract class DismissedReview implements Parcelable {
+        @Json(name = "review_id")
+        @Nullable
+        public abstract Integer reviewId();
+
+        @Nullable
+        public abstract ReviewState state();
```

Comment:
This seems off:

```
[...]
    "dismissed_review": {
      "state": 30,
      "review_id": 73754698,
      "dismissal_message": "Testing dismissing",
      "dismissal_commit_id": null
    }
  }
[...]
```

I've asked Github support for clarification; they said they'll come back to me.